pyproject.toml

```
@@ -8,14 +8,14 @@ readme = "README.md"
 repository = "https://github.com/palaimon/ipyannotator"
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = ">=3.8,<3.11"
```

```suggestion
python = ">=3.8"
```

[Never cap the Python version number](https://iscinumpy.dev/post/bound-version-constraints/). You are almost assuredly not doing what you think and are making it impossible for people to use your software in the future.